Cabal/Distribution/Simple/Configure.hs

```diff
@@ -146,6 +146,7 @@ import Distribution.Compat.Environment ( lookupEnv )
 import Distribution.Compat.Exception ( catchExit, catchIO )
 
 import qualified Data.Set as Set
+import qualified Prelude (IO)
```

Comment:
We have

```
type NoCallStackIO a = OrigPrelude.IO a
```

defined in `Distribution.Compat.Prelude`, please don't import `Prelude (IO)`.